climada/entity/exposures/crop_production.py

```diff
@@ -366,7 +434,20 @@ def set_mean_of_several_models(self, input_dir=INPUT_DIR, hist_mean=HIST_MEAN_PA
 
         return self
 
-    def set_to_usd(self, input_dir=INPUT_DIR, yearrange=YEARS_FAO):
+    def set_to_kcal(self):
+        """Converts the exposure from tonnes to kcal using conversion factor per crop type.
+
+        Returns:
+            Exposure
+        """
+        if not 't' in self.value_unit:
```

Comment:
What if a unit which contains the letter `t` but is different from `t` or `t/y` is set by a user?

Reply:
There is some confusion between "tonnes" and "t" and "t / y", ... I will clean this up.
However, if the user sets a unit manually, that's probably intentional and assumed to be for a reason, can't catch that.